src/constraints/simplex.rs

```diff
@@ -0,0 +1,80 @@
+use super::Constraint;
+
+#[derive(Copy, Clone)]
+/// A simplex with alpha 'a', that is,
```

Comment:
We can use LaTeX expressions in the documentation ([example](https://docs.rs/optimization_engine/0.7.2/optimization_engine/constraints/index.html)).